sentry/templates/configmap-sentry.yaml

```diff
@@ -187,18 +179,26 @@ data:
     # General #
     ###########
 
-
-    secret_key = env('SENTRY_SECRET_KEY')
-    if not secret_key:
-      raise Exception('Error: SENTRY_SECRET_KEY is undefined')
-
-    SENTRY_OPTIONS['system.secret-key'] = secret_key
-
     # Instruct Sentry that this install intends to be run by a single organization
     # and thus various UI optimizations should be enabled.
     SENTRY_SINGLE_ORGANIZATION = {{ if .Values.sentry.singleOrganization }}True{{ else }}False{{ end }}
 
-    SENTRY_OPTIONS["system.event-retention-days"] = int(env('SENTRY_EVENT_RETENTION_DAYS') or {{ .Values.sentry.cleanup.days | quote }})
+    SENTRY_OPTIONS["system.event-retention-days"] = int(
+        env('SENTRY_EVENT_RETENTION_DAYS') or {{ .Values.sentry.cleanup.days | quote }}
+    )
+
+    #########
+    # Redis #
+    #########
+    
+    # Generic Redis configuration used as defaults for various things including:
+    # Buffers, Quotas, TSDB
+
+    SENTRY_OPTIONS["redis.clusters"] = {
```

Comment:
added redis config here to match

[reference](https://github.com/getsentry/self-hosted/blob/master/sentry/sentry.conf.example.py#L73)